sdk/src/test/java/com/mapbox/maps/renderer/RenderHandlerThreadTest.kt

```diff
@@ -70,7 +59,23 @@ class RenderHandlerThreadTest {
     Shadows.shadowOf(Looper.getMainLooper()).pause()
     renderHandlerThread.post { action() }
     Shadows.shadowOf(Looper.getMainLooper()).idle()
-    verify(exactly = 0) { action.invoke() }
+    verifyNo { action.invoke() }
+  }
+
+  @Test
+  fun postThreadStopped() {
+    val actionOne = mockk<() -> Unit>(relaxed = true)
+    val actionTwo = mockk<() -> Unit>(relaxed = true)
+    Shadows.shadowOf(Looper.getMainLooper()).pause()
+    renderHandlerThread.apply {
+      start()
+      post { actionOne() }
+      stop()
+      post { actionTwo() }
```

Comment:
should we check also `postDelayed` with some delay that happens after `stop`?
and check thread queue.

Reply:
Ok with the delay, not sure it's worth to check the queue directly though